CardPayments/src/main/java/com/paypal/android/cardpayments/CardClient.kt

```diff
@@ -42,15 +42,13 @@ class CardClient internal constructor(
     private val lifeCycleObserver = CardLifeCycleObserver(this)
 
     private val approveOrderExceptionHandler = CoreCoroutineExceptionHandler { error ->
-        notifyApproveOrderFailure(error)
+        notifyApproveOrderFailure(error, null)
     }
 
     private val vaultExceptionHandler = CoreCoroutineExceptionHandler { error ->
         cardVaultListener?.onVaultFailure(error)
     }
 
-    private var orderId: String? = null
```

Comment:
It feels like a code smell saving `orderId` in a member variable since they never get cleared. I've removed these internal state variables where possible.